test/unit/services/parser-service.test.js

```diff
@@ -311,6 +311,275 @@ describe('parseAsda', () => {
   })
 })
 
+// Tesco1
+
+describe('matchesTescoModel1', () => {
+  test('returns true', () => {
+    const filename = 'PackingListTesco1.xlsx'
+    const packingListJson = {
+      Input_Data_Sheet: [
+        {},
+        {},
+        {},
+        {
+          AT: 'RMS-GB-000022-998'
+        },
+        {
+          G: 'Product/ Part Number description',
+          L: 'Tariff Code UK',
+          AS: 'Treatment Type',
+          AT: 'Green Lane',
+          BR: 'Packages',
+          BT: 'Gross Weight',
+          BU: 'Net Weight'
+        }
+      ]
+    }
+    const result = parserService.matchesTescoModel1(packingListJson, filename)
+    expect(result).toBeTruthy()
+  })
+})
+
+test('returns false for empty json', () => {
+  const packingListJson = {}
+  const filename = 'packinglist.xls'
+  const result = parserService.matchesTescoModel1(packingListJson, filename)
+  expect(result).toBeFalsy()
+})
+
+test('returns false for missing establishment number', () => {
+  const packingListJson = {
+    Input_Data_Sheet: [
+      {},
+      {},
+      {
+        I: 'INCORRECT'
+      }
+    ]
+  }
+  const filename = 'packinglist.xls'
```

Comment:
filename is wrong, I think its failing before it even gets to the check for the establishment number